spec/spec_helper.rb

```diff
@@ -1,8 +1,15 @@
 require 'rspec'
 require "signable"
+require "support/vcr"
+require "support/webmock"
+require 'pry'
 
 Dir.glob("#{File.dirname(__FILE__)}/support/**/*.rb").each { |f| require f }
 
-RSpec.configure do |c|
-  c.mock_with :rspec
+RSpec.configure do |config|
+  config.mock_with :rspec
+
+  config.around(:each, :vcr) do |example|
+    VCR.use_cassette(*example.metadata[:vcr]) { example.call }
```

Comment:
```suggestion
    VCR.use_cassette(*{ re_record_interval: 1.month }.merge(example.metadata[:vcr])) { example.call }
```
was thinking it might be good to enforce re-recording of the interactions to ensure they dont go stale

Reply:
That syntax did not work because `example.metadata[:vcr]` is a String, not a Hash, so I've done it like this instead:
`VCR.use_cassette(example.metadata[:vcr], re_record_interval: 1.month) { example.call }`